blues/simulation.py

```diff
@@ -484,6 +501,20 @@ def simulateMD(self, nstepsMD=5000, **opt):
         self.nc_context.setPositions(md_state0['positions'])
         self.nc_context.setVelocities(md_state0['velocities'])
 
+    def _report(self,start,nc_step):
+        end = time.time()
+        headers = ['Step', 'Speed (ns/day)', 'Acc. Moves', 'Iter']
+        if self.current_iter == 0 and nc_step == 0:
+            self.log.info('[NCMC] "%s"' % ('"'+'\t'+'"').join(headers))
+        if nc_step % self.opt['reporter_interval'] == 0 or nc_step+1 == self.opt['nstepsNC']:
+            elapsed = end-start
+            elapsedDays = (elapsed/86400.0)
+            elapsedNs = (self.nc_context.getState().getTime()-self._initialSimulationTime).value_in_unit(unit.nanosecond)
+            speed = (elapsedNs/elapsedDays)
+            speed = "%.3g" % speed
```

Comment:
Do you want to indicate that the speed is in ns/day in the output?

Reply:
It is indicated in the headers.